drivers/at/at.c

```diff
@@ -334,132 +388,129 @@ ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command,
 
     res = at_send_cmd(dev, command, timeout);
     if (res) {
-        goto out;
+        return res;
     }
 
     memset(resp_buf, '\0', len);
 
-    while (1) {
-        res = at_readline(dev, pos, bytes_left, keep_eol, timeout);
+    bool first_line = true;
+    while (bytes_left) {
+        if (first_line) {
+            res = at_readline_skip_empty(dev, pos, bytes_left, keep_eol, timeout);
+            first_line = false;
+        } else {
+            // keep subsequent empty lines, for whatever reason
```

Comment:
```suggestion
            /* keep subsequent empty lines, for whatever reason */
```

let's stay with one comment style